### ISSUE TITLE
Font size in `nvim-qt` and `nvy` differs

### ISSUE THREAD

**Issue by VladimirMarkelov** [RMichelsen/Nvy#1]
![image](https://user-images.githubusercontent.com/1577186/94187105-5bdedd00-fe5c-11ea-8c7d-6e59172880f7.png)

It is for `set guifont=Cascadia\ Mono\ PL\ SemiLight:h12`. For `h14` the difference seems more noticeable.

If fixing text display size is not easy, could you set `nvim` variable at start, so I can create `if` block in `init.vim` to set specific configuration just for `nvy`?

**Comment by VladimirMarkelov** [comment]
The variable works like charm. Thanks! JFY, I failed to make sizes of font the same in both applications. `h10` makes font smaller than in `nvim-qt` and `h11` is a bit bigger. I tried fractional heights(vim helps says it is OK), but it did not work for me as expected(`10.2` looked like `10`, while `10.3` jumped in size and looked like `11`). So, I left it at `11`.

**Comment by VladimirMarkelov** [comment]
Another piece of information: I managed to make fonts in `nvim-qt` and `nvy` look the same (while I used size `14` for `nvim-qt` and `10` for `nvy`). The trouble was that I used variable font and used a virtual name `Cascadia Mono PL SemiLight` (the real font name is `Cascadia Mono PL` that have 6 different weights selected with extra word after the name. `nvim-qt` seems to understand virtual names, but `nvy` fell back to default font, hence I was not able to make text the same size - it is not generally possible when using different fonts :)  So, I removed variable font and installed two separate 'regular' and 'bold', now both `nvim-qt` and `nvy` uses the same font and I managed to set up them to look the same.

Sorry for extra confusion. Anyway, without new `g:nvy` variable I wouldn't be able to do it. Thanks again!
